README.md

```diff
@@ -1,146 +1,195 @@
     </activity>
 ```
 
-### Step 2: Instantiate MSAL and Acquire a Token
+### Step 4: Create an MSAL PublicClientApplication
 
-1.  Create a new PublicClientApplication instance. Make sure to fill in your app/client id
+>NOTE: In this example were creating an instance of MultipleAccountPublicClientApplication which is designed to work with apps that allow multiple accounts to be used within the same application.  For more information on multiple vs. single account public client applications click [here](link)
 
-```Java
-    PublicClientApplication myApp = new PublicClientApplication(
-                    this.getApplicationContext(),
-                    R.raw.auth_config);
-```
-
-2. Acquire a token
+1.  Create a new MultipleAccountPublicClientApplication instance. 
 
 ```Java
-    myApp.acquireToken(this, SCOPES, getAuthInteractiveCallback());
-```
 
-### Step 3: Configure the Auth helpers
+String[] scopes = {"User.Read"};
+IMulitipleAccountPublicClientApplication mMultipleAccountApp = null;
+IAccount mFirstAccount = null;
 
-1. Create an onActivityResult method
+PublicClientApplication.createMultipleAccountPublicClientApplication(getContext(),
+    R.raw.msal_config,
+    new IPublicClientApplication.IMultipleAccountApplicationCreatedListener() {
+        @Override
+        public void onCreated(IMultipleAccountPublicClientApplication application) {
+            mMultipleAccountApp = application;
+        }
 
-```Java
-    /* Handles the redirect from the System Browser */
-    @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        sampleApp.handleInteractiveRequestRedirect(requestCode, resultCode, data);
-    }
+        @Override
+        public void onError(MsalException exception) {
+            //Log Exception Here
+        }
+    });
 ```
 
-2. Create the getAuthInteractiveCallback method
+2. Acquire a token interactively
 
-```Java
-    private AuthenticationCallback getAuthInteractiveCallback() {
-        return new AuthenticationCallback() {
-            @Override
-            public void onSuccess(AuthenticationResult authenticationResult) {
-                /* Successfully got a token, use it to call a protected resource */
+```java
 
-                String accessToken = authenticationResult.getAccessToken();
-            }
-            @Override
-            public void onError(MsalException exception) {
-                /* Failed to acquireToken */
-
-                if (exception instanceof MsalClientException) {
-                    /* Exception inside MSAL, more info inside MsalError.java */
-                } else if (exception instanceof MsalServiceException) {
-                    /* Exception when communicating with the STS, likely config issue */
-                }
-            }
-            @Override
-            public void onCancel() {
-                /* User canceled the authentication */
+mMultipleAccountApp.acquireToken(this, SCOPES, getAuthInteractiveCallback());
```

Comment:
should we also add how to `acquireToken` with `AcquireTokenParameters` ?

Reply:
I used the `Parameters` objects _exclusively_ in my B2C doc